pkgs/top-level/all-packages.nix

```diff
@@ -4789,6 +4789,8 @@ with pkgs;
 
   sov = callPackage ../tools/wayland/sov { };
 
+  sway-audio-idle-inhibit = callPackage ../tools/wayland/sway-audio-idle-inhibit { };
+
```

Comment:
Using `pkgs/by-name` is now recommended https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md